gameday/dto.go

```diff
@@ -7,6 +7,29 @@ import (
 	"time"
 )
 
+// ConfigureDTO the data transfer object for
+// configure application database
+type ConfigureDTO struct {
+	Scheme string `json:"scheme"`
+	Url    string `json:"url"`
+}
+
+// Validate check if the DTO has the required values
+func (c ConfigureDTO) Validate() error {
+	if c.Scheme == "" {
+		return errors.New("failed: missing required field `scheme`")
+	}
+
+	if c.Scheme != "sqlite" && c.Scheme != "sqlite3" && c.Scheme != "postgresql" && c.Scheme != "postgres" {
+		return errors.New("failed: given value for `scheme` is not supported")
```

Comment:
let's create an error type and return `ErrInvalidDatabaseScheme`, so we can be able in tests to assert this accordingly